chef/cookbooks/provisioner/templates/default/tftp.erb

```diff
@@ -8,8 +8,8 @@ service tftp
        socket_type             = dgram
        protocol                = udp
        wait                    = yes
-       flags                   = IPv6 IPv4
+       flags                   = IPv4
```

Comment:
Bug now has more info. There is a memory overflow in tftpd because IPv4 addrs are mapped to IPv6-addrs like ::ffff:123.123.123.123 which are much longer when our \x expansion is applied for hyper-v booting